base/build.gradle.kts

```diff
@@ -11,4 +11,8 @@ dependencies {
     implementation(libs.compose.material)
 
     debugImplementation(libs.compose.ui.tooling)
+
+    // Temporary fix for preview bug https://issuetracker.google.com/issues/227767363
```

Comment:
<img width="406" alt="image" src="https://user-images.githubusercontent.com/16294951/177183581-6603474e-1723-4a9d-a05d-d4745dca1dfe.png">

could you please check on Dolphin?

Reply:
Yes, it's Chipmunk's issue. I've added todo to not forget to remove these dependencies when fix come out.